src/basic/fs-util.c

```diff
@@ -432,13 +432,21 @@ int symlink_idempotent(const char *from, const char *to, bool make_relative) {
         return 0;
 }
 
-int symlink_atomic(const char *from, const char *to) {
-        _cleanup_free_ char *t = NULL;
+int symlink_atomic_full(const char *from, const char *to, bool make_relative) {
+        _cleanup_free_ char *relpath = NULL, *t = NULL;
         int r;
 
         assert(from);
         assert(to);
 
+        if (make_relative) {
+                r = path_make_relative_parent(to, from, &relpath);
```

Comment:
Why use `path_make_relative_parent()` here instead of `path_make_relative()`? I feel like this deserves a comment

Reply:
E.g. to: /path/to/aaa and from: /path/to/bbb, then
- path_make_relative(to, from) -> "../aaa"
- path_make_relative_parent(to, from) -> "aaa"

I will comment that somewhere in another PR.